README.md

```diff
@@ -244,9 +244,7 @@ The `upload()` method requires at least the `file` and the `fileName` parameter
 
 Sample usage
 ```cs
-ImagekitResponse resp = await imagekit
-    .FileName("my_file_name.jpg")
-    .UploadAsync(<fullPath|url|base_64|binary>);
+ResponseMetaData resp = await imagekit .UploadAsync(<fullPath|url|base_64|binary>,"my_file_name.jpg");
```

Comment:
Is every method response is of type `ResponseMetaData`? 

Also the space ` `  between `imagekit` and `UploadAsync ` seems to be incorrect.

Show an example similar to what we have in Node.js SDK.

Reply:
Yes, ResponseMetaData common object used to hold response.